src/libs/SidebarUtils.ts

```diff
@@ -221,7 +222,13 @@ function getOptionData({
         isDeletedParentAction: false,
     };
 
-    const participantPersonalDetailList = Object.values(OptionsListUtils.getPersonalDetailsForAccountIDs(report.participantAccountIDs ?? [], personalDetails)) as PersonalDetails[];
+    let participantAccountIDs = report.participantAccountIDs ?? [];
+
```

Comment:
Can you add a comment for why we're doing this? same with the other spot

Reply:
@thienlnam was there a reason to not include the `currentUser` in the `report.participantAccountIDs`? We're using this hack in multiple places as a reason.